### ISSUE TITLE
Nginx missing after upgrade, v 1.0

### ISSUE THREAD

**Issue by ManfredLan** [plesk/centos2alma#122]
**Describe the bug**
For whatever reason the centos2alma script does not install nginx during the stage of reinstalling Plesk components, so websites are not available.
It is required to manually install, disable and then enable nginx afterwards.

**Feedback archive**
[centos2alma_feedback(1).zip](https://github.com/plesk/centos2alma/files/12103811/centos2alma_feedback.1.zip)

**Comment by SandakovMM** [comment]
Hello,
Thank you for reaching out to us with this issue.
I noticed that sw-nginx was removed by leapp during the conversion process:
```log
2023-07-19 10:02:46,917 - INFO - yum-metadata-parser                    x86_64  1.1.4-10.el7                                  @System                         57 k
2023-07-19 10:02:46,917 - INFO - yum-plugin-versionlock                 noarch  1.1.31-54.el7_8                               @System                         53 k
2023-07-19 10:02:46,917 - INFO - Removing dependent packages:
2023-07-19 10:02:46,917 - INFO - ImageMagick                            x86_64  6.9.10.68-6.el7_9                             @System                        7.9 M
2023-07-19 10:02:46,917 - INFO - awstats                                noarch  7.8-3.el7                                     @System                        7.5 M
...

2023-07-19 10:02:47,105 - INFO - sw-libboost-regex1.54.0                x86_64  1.54.0-14062011                               @System                        1.0 M
2023-07-19 10:02:47,106 - INFO - sw-libboost-regex1.55.0                x86_64  1.55.0-centos7.14070415                       @System                        1.0 M
**2023-07-19 10:02:47,106 - INFO - sw-nginx                               x86_64  1.24.0-2.centos.7+p18.0.53.0+t230516.1254     @System                         16 M**
2023-07-19 10:02:47,106 - INFO - uptrack                                noarch  1.2.75-0.el7                                  @System                        525 k
2023-07-19 10:02:47,106 - INFO - yum-plugin-changelog                   noarch  1.1.31-54.el7_8                               @System                         40 k
```
It seems that the problem occurred because dnf recognized the sw-nginx package as installed from the System repository, whereas it should have been installed from PLESK_18_0_53-dist. As a result, when leapp tried to find an analogous package in the alma system repository, it failed. Unfortunately, I am unable to determine the exact reason for this behavior or reproduce it on my side.
However, we can work around this issue by reinstalling nginx after the conversion is done. I'll propose the change later this week.

**Comment by SandakovMM** [comment]
I've merged nginx re-installation (PR #123). It should help to avoid the problem after conversion.
